ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml

```diff
@@ -53,6 +53,13 @@ tests:
   optional: true
   steps:
     workflow: openshift-ci-security
+- as: fips-image-scan-service
+  skip_if_only_changed: ^docs/|\.md$|lightspeed-catalog.*|^\.tekton/|^\.github/|^hack/|^(?:.*/)?(?:\.gitignore|\.snyk|\.dockerignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
service repository does not have these files `lightspeed-catalog.*` but it does not harm